### PR TITLE
 #169817534 chore(coveralls):install and integrate coveralls

### PR DESCRIPTION
**Pull request by victor-abz**
#### What does this PR do?
`integrate coveralls` 
`correct Github links.`
#### Description of Task to be completed?
`integrate coveralls with Github, and install it in nod js`
#### How should this be manually tested?
`we will test after installing Travis because we can't get coverage before Travis integration`
#### What are the relevant pivotal tracker stories?
`#169817534`
#### Questions:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,27 @@
 {
-  "name": "blackninjas-backend",
+  "name": "the-snipers-backend",
   "version": "1.0.0",
   "description": "A platform to make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "NODE_ENV=test nyc --reporter=text --reporter=html ./node_modules/.bin/_mocha ./src/test/**.test.js ",
```
Comment (adesege) [inline package.json]: You don't need the test script here

File: package.json [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
+  "author": "theSnippers",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/andela/blackninjas-backend/issues"
+    "url": "https://github.com/andela/the_spinners-backend/issues"
   },
+  "homepage": "https://github.com/andela/the_spinners-backend#readme",
+  "devDependencies": {
+    "coveralls": "^3.0.9",
+    "mocha": "^6.2.2",
```
Comment (adesege) [inline package.json]: Remove packages that are not used in this task

**Review by adesege — CHANGES_REQUESTED** [review]
Well done, Victor. My comments are interspersed below.

File: package.json (hunk 2 above)
Comment (victor-abz) [reply package.json]: Okay let me do it

File: package.json (hunk 1 above)
Comment (victor-abz) [reply package.json]: Ok

**Pull request merged**